cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -42,19 +42,23 @@ endmacro()
 
 boost_component(atomic 1.53.0)
 boost_component(chrono 1.47.0)
+boost_component(compute 1.61.0)
```

Comment:
We don't need to list header-only components here. The goal is to have "not building" everywhere when Boost header-only part is building. For now it looks like this (see [log](https://travis-ci.org/ingenue/hunter/jobs/130565757#L984)):
```
Component configuration:

    - atomic                   : not building
    - chrono                   : not building
    - container                : not building
    - context                  : not building
    - coroutine                : not building
    - coroutine2               : not building
    - date_time                : not building
    - exception                : not building
    - filesystem               : not building
    - graph                    : not building
    - graph_parallel           : not building
    - iostreams                : not building
    - locale                   : not building
    - log                      : not building
    - math                     : not building
    - metaparse                : building
    - mpi                      : not building
    - program_options          : not building
    - python                   : not building
    - random                   : not building
    - regex                    : not building
    - serialization            : not building
    - signals                  : not building
    - system                   : not building
    - test                     : not building
    - thread                   : not building
    - timer                    : not building
    - type_erasure             : not building
    - wave                     : not building
```

I.e. you only need to add `boost_component(metaparse 1.61.0)`